crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -1192,8 +1228,8 @@ impl<'db> TypeInferenceBuilder<'db> {
                             "invalid-context-manager",
                             format_args!("
                                 Object of type `{context_expression}` cannot be used with `with` because the method `__enter__` of type `{enter_ty}` is not callable",
-                                context_expression = context_expression_ty.display(self.db),
-                                enter_ty = enter_ty.display(self.db)
+                                         context_expression = context_expression_ty.display(self.db),
+                                         enter_ty = enter_ty.display(self.db)
```

Comment:
Weird, why isn't rustfmt formatting deterministic here and below?

Reply:
(It's because it's within a macro, I think.)

Reply:
(Oh sorry -- not sure about below, but here yeah.)